.github/dependabot.yml

```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "weekly"
```

Comment:
Do you think a weekly rate is good? I think it's too often for a project. Maybe we can change it to `monthly`?

Reply:
@hpneo makes sense.
I updated it